geode-common/src/main/java/org/apache/geode/util/internal/GeodeJsonMapper.java

```diff
@@ -34,13 +36,21 @@ public class GeodeJsonMapper {
    */
   public static ObjectMapper getMapper() {
     ObjectMapper mapper = JsonMapper.builder()
+        .addModule(new JavaTimeModule())
```

Comment:
Now I think of it, we should not spell out those modules here. Instead, we should do `findAndRegisterModules` to search for all modules available on the classpath. This would get rid of the need to require these on the management java client. I will submit a commit for iit.

Reply:
it's still available and working on Jackson 2.13. I didn't find the method `findAndAddModules`

Reply:
On a 2nd though, since these `getMapper` are called on each query/get/function execution, `findModule` is not cached and could be expensive, so using this might not be a good idea. It's fine that our management java client also supports joda time as well.